tests/testapp/models.py

```diff
@@ -23,5 +23,5 @@ class TestParentModel(models.Model):
 
 
 class TestChildModel(models.Model):
-    parent = models.ForeignKey(TestParentModel)
+    parent = models.ForeignKey(TestParentModel, on_delete=models.CASCADE)
```

Comment:
`on_delete` is always required.

https://docs.djangoproject.com/en/2.0/releases/1.9/#foreignkey-and-onetoonefield-on-delete-argument